bentoml/cli/__init__.py

```diff
@@ -104,6 +107,9 @@ def info(archive_path=archive_path):
         List all APIs defined in the BentoService loaded from archive
         """
         model_service = load(archive_path)
+
+        track_cli('info', model_service)
```

Comment:
no need to do this in every command, just add it to the command group definition, which will be invoked for every click command